config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/tasks/zip/builder/ServiceEntityBuilder.java

```diff
@@ -50,16 +51,17 @@ public Integer getOrder() {
         return ORDER;
     }
 
-    private Entity buildServiceEntity(Bundle bundle, String policyPath, Service service, Document document) {
-        Policy policy = bundle.getPolicies().get(policyPath);
+    private Entity buildServiceEntity(Bundle bundle, String name, Service service, Document document) {
+        String processedName = name.lastIndexOf(File.separatorChar) == -1 ? name : name.substring(name.lastIndexOf(File.separatorChar) + 1);
```

Comment:
try using here `FilenameUtils.getBaseName` from commons-io, you probably can get rid of this conditional :)

Reply:
oh thanks for that, will use FilenameUtils.getBaseName